src/libs/translations/index.ts

```diff
@@ -0,0 +1,8 @@
+import HttpBackend from 'i18next-http-backend';
```

Comment:
cant you export directly?

`export HttpBackend from "i18next-http-backend"`

Reply:
Yes, wanted to import as default and name export so I used that:
export { default as HttpBackend } from 'i18next-http-backend';

Applied to all imports in the file.